tools/StaticAnalysis/Exceptions/SignatureIssues.csv

```diff
@@ -1453,3 +1453,18 @@
 "Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureRmIpsecPolicyCommand","New-AzureRmIpsecPolicy","1","8410","Parameter SADataSizeKilobytes of cmdlet New-AzureRmIpsecPolicy does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureVirtualNetworkGatewayConnectionCommand","New-AzureRmVirtualNetworkGatewayConnection","1","8410","Parameter UsePolicyBasedTrafficSelectors of cmdlet New-AzureRmVirtualNetworkGatewayConnection does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
 "Microsoft.Azure.Commands.Network.dll","Microsoft.Azure.Commands.Network.NewAzureVirtualNetworkGatewayConnectionCommand","New-AzureRmVirtualNetworkGatewayConnection","1","8410","Parameter IpsecPolicies of cmdlet New-AzureRmVirtualNetworkGatewayConnection does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"Microsoft.Azure.Commands.AnalysisServices.Dataplane.dll","Microsoft.Azure.Commands.AnalysisServices.Dataplane.RestartAzureAnalysisServer","Restart-AzureAnalysisServicesInstance","2","8210","Restart-AzureAnalysisServicesInstance does not have a Force parameter but the cmdlet verb 'Restart' indicates that it may perform destructive actions under certain circumstances. Consider whether the cmdlet should have a Force parameter anduse ShouldContinue under some circumstances. ","Consider wether the cmdlet should have a Force parameter and use ShouldContinue under some circumstances. "
```

Comment:
You don't need to set suppressions for severity 2 signature issues - these are just wranings.  Sev 0 and Sev 1 need to be suppressed, if valid.